attributes/default.rb

```diff
@@ -88,3 +88,7 @@
 # <chef_cache_path>/cookbooks/audit/inspec-<YYYYMMDDHHMMSS>.json
 default['audit']['json_file']['location'] =
   File.expand_path(Time.now.utc.strftime('../../inspec-%Y%m%d%H%M%S.json'), __FILE__)
+
+# Controls that have a run_time below this limit will be stripped of the `start_time` and
+# `start_time` fields to reduce the size of the reports being sent to Automate
```

Comment:
nitpick: did you mean start time and end time?

Reply:
does this mean the report wouldnt have an end time? is that ok with automate ingestion?

Reply:
These are for the control results. Let me rephrase the comment to be more clear about this:

```
# Control results that have a `run_time` below this limit will be stripped of the 
# `start_time` and `run_time` fields to reduce the size of the reports being sent to Automate
```